### PR TITLE
Wolfe/docs nav

### PR DESCRIPTION
**Pull request by alexwolfe**
ok @benarent this should fix the navigation issue for mobile. Please take a look, thanks!

### CODE REVIEW

File: docs/4.3/faq.md [hunk 1]
```diff
@@ -52,9 +52,9 @@ Scenario | Max Recommended Count | Proxy | Auth server
 Teleport nodes connected to auth server | 10,000 |2x  2-4 vCPUs/8GB RAM | 2x 4-8 vCPUs/16GB RAM
 Teleport nodes connected to proxy server (IoT) | 2,000* | 2x 2-4 vCPUs/8GB RAM |2x 4-8 vCPUs/16+GB RAM
 
-### Which version of Teleport is supported?
+## Which version of Teleport is supported?
 
-| Release       | LTS | Release Date         | Supported Until      | Minimum tsh version  |
+| Release       | Long Term Support | Release Date         | Supported Until      | Min tsh version  |
```
Comment (webvictim) [inline docs/4.3/faq.md:57]: Why did we change from `Minimum` to `Min`?

Comment (alexwolfe) [reply docs/4.3/faq.md:57]: I felt that we could shorten it to help these items display on screen better while not sacrificing a user's understanding of the title.

**Comment by benarent** [comment]
This does fix this bug
image


but we should review our TOC for large pages. e.g. https://share.getcloudapp.com/o0uvzoOL

File: docs/4.3/faq.md (hunk 1 above)
Comment (benarent) [reply docs/4.3/faq.md:57]: I checked this on my phone, and we should make these tables scroll on mobile.
image

**Review by benarent — APPROVED** [review]
Let's merge this in, but address the other issues in another PR.

**Pull request merged**